SentryLogRoute.php

```diff
@@ -21,10 +24,31 @@ class SentryLogRoute extends CLogRoute
 
 	/**
 	 * Raven_Client instance from SentryComponent->getRaven();
-	 * @var Raven_Client
+	 * @var ClientInterface
 	 */
 	protected $raven;
 
+	public static function getSeverityFromLogLevel(string $level): Severity
+	{
+		$severityLevels = [
+			CLogger::LEVEL_PROFILE => Severity::debug(),
+			CLogger::LEVEL_TRACE   => Severity::debug(),
+			'debug'                => Severity::debug(),
+			CLogger::LEVEL_INFO    => Severity::info(),
+			CLogger::LEVEL_WARNING => Severity::warning(),
+			'warn'                 => Severity::warning(),
+			CLogger::LEVEL_ERROR   => Severity::error(),
+			'fatal'                => Severity::fatal(),
```

Comment:
```suggestion
			'fatal'                => Severity::error(),
```

Reply:
Discussed this with Mike, and wanted to record why this is the way that it is.

The initial concern is that we are standardizing our logging in the application to 3 main levels (`info`, `warning` & `error`) with `debug` being used sparingly. This particular line introducing the `fatal` level seems to run counter to that standardization. 

However, this package is isolated from that standardization and only aims to provide a way of mapping the Yii logs to a Sentry format. Since Sentry provides the `fatal` level, I included it as an option here, even if we _shouldn't_ be using it from within our application code. The standardization concern is something for the consumer of this package (`skillshare/skillshare`)